docs/admin_api/media_admin_api.md

```diff
@@ -100,3 +100,27 @@ Response:
   "num_quarantined": 10  # The number of media items successfully quarantined
 }
 ```
+
+# Draft: Delete local media
+This API deletes the *local* media from the disc of your own server.
+
+Request:
+
+```
+DELETE /_synapse/admin/v1/media/<server_name>/<media_id>
+
+{}
+```
+
+URL Parameters
+
+* `server_name` - The name of your local server (e.g `matrix.org`)
+* `media_id` - The ID of the media (e.g `abcdefghijklmnopqrstuvwx`)
+
+Response:
+
+```
+{
+  "deleted": 1  # The number of media items successfully deleted
+}
```

Comment:
This is a bit useless if we're only ever going to be deleting one or zero files. Ideally we'd return `{}` if the delete was a success, and instead a non-200 error if there was an issue.

Are you planning to add an endpoint in the future that allows you to delete more than one media file at a time? If so, why not do that now as presumably this API would be less useful when that comes out?

Reply:
My idea was to create a small PR. I can add more functions in this PR.